content/docs/introduction/faq.md

```diff
@@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
 Yes, with [PromDash](/docs/visualization/promdash/) and [Console
 templates](/docs/visualization/consoles/).
 
+### Can I change the timezone? Why is everything in UTC?
+
+For monitoring purposes, it is absurd to use any timezone observing
+so-called daylight saving time. To avoid any kind of timezone
+confusion, we decided to exclusively use UTC for Prometheus. For pure
```

Comment:
Yeah, it's unixtime internally and UTC for display purposes. The FAQ is probably asked because somebody saw UTC.